jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -184,6 +184,10 @@ class java_bytecode_languaget:public languaget
   class_hierarchyt class_hierarchy;
   // List of classes to never load
   std::unordered_set<std::string> no_load_classes;
+
+  // friend function to access pointer type selector in unit tests
+  friend const select_pointer_typet &
+  get_pointer_type_selector_unit(const java_bytecode_languaget &);
```

Comment:
Is the function definition part of some other PR?

Reply:
yes, it had planned to make it live in TG, but I could also add it to jbmc unit `java-testing-utils`

Reply:
If it would only be in TG but not in jbmc (which is open source), I'm thinking people might be tempted to delete it maybe because they won't see the reason for it. What do you think?